src/kernels/jupyter/types.ts

```diff
@@ -234,11 +234,12 @@ export interface IJupyterUriProviderRegistration {
     getJupyterServerUri(id: string, handle: JupyterServerUriHandle): Promise<IJupyterServerUri>;
 }
 
+// An entry into our list of saved servers
 export interface IJupyterServerUriEntry {
-    uri: string;
-    serverId: string;
-    time: number;
-    displayName?: string;
+    uri: string; // URI of the server to connect to
+    serverId: string; // Unique ID using a hash of the full uri
+    time: number; // The most recent time that we connected to this server
```

Comment:
Can we add jsDoc style comments
```suggestion
    // The most recent time that we connected to this server
    time: number; 
```

```suggestion
    /** 
     * The most recent time that we connected to this server
     */
    time: number; 
```

These comments show up better in typescript. Else one would have to use `go to definition` to see the above comments.

Reply:
Bleh, I agree in this case for the definition look up, so I'll change 'em. But I never like how much space the jsdoc style takes up vertically.

Reply:
Update incoming